CHANGES.rst

```diff
@@ -11,6 +11,8 @@ New Features
 - The line analysis plugin now includes logic to account for the background
   continuum. [#1060]
 
+- Specviz can load a ``SpectrumList`` and combine all its elements into a single spectrum. [#1014]
+
```

Comment:
Can we update the Sphinx documentation to include some fleshed out examples of the different types of input into `load_spectrum` and how to load data?